docs/release.rst

```diff
@@ -14,6 +17,8 @@ Next release
 * Fix minor bug in `N5Store`. 
   By :user:`gsakkis`, :issue:`550`.
 
+* add new `FSStore` to open any ``fsspec`` supported protocol :issue:`546`
+
```

Comment:
Looks like this actually shows up below. So maybe we can drop it?

Reply:
```suggestion
```